2_process/src/process_model_data.r

```diff
@@ -1,11 +1,6 @@
-process_data <- function() {
-  # Set file paths for data
-  project_input_dir <- '1_fetch/out'
-  project_output_dir <- '2_process/out'
-  # Grab the data
-  mendota_file <- file.path(project_input_dir, 'model_RMSEs.csv')
-  # Prepare the data for plotting
-  eval_data <- readr::read_csv(mendota_file, col_types = 'iccd') %>%
+process_data <- function(project_input_dir) {
+    # Prepare the data for plotting
+  eval_data <- readr::read_csv(project_input_dir, col_types = 'iccd') %>%
```

Comment:
Nice! Good catch that the incoming data file will need to be read in, since `model_RMSEs_csv` is  file target